ynput/OpenPype
# Hiero: colorspace settings aligned with nuke - AY-978

## Changelog Description
In order to share the same colorspaces in the workfile in Hiero and Nuke, we need to bring back the workfile settings for colorspaces in Nuke.

In Hiero we also need code to edit the project settings in memory and apply the colorspaces when launching Hiero so any new project gets the correct colorspaces. Due to Foundry not providing Python API methods for setting the project colorspaces, we need to go through the UI widgets to set them, when dealing with in-memory projects.

Also small bugfix when saving the workfile without any sequences.

## Testing notes:
1. Reset color management settings for Global, Nuke and Hiero.
2. Enable Global color management; `project_settings/global/imageio/activate_global_color_management`
3. Launch Nuke and Hiero.
4. Verify the colorspaces are setup correctly.
